File: ambientes/src/NumPrimo.java
import java.util.Scanner;
public class NumPrimo {
    public void verNumPrimo(){
        Scanner scanner = new Scanner(System.in);
        System.out.print("Digite um número: ");
        int num = scanner.nextInt();
        scanner.close();

        boolean primo = true;

        if (num < 2) {
            primo = false;
        } else {
            for (int i = 2; i <= Math.sqrt(num); i++) {
                if (num % i == 0) {
                    primo = false;
                    break;
                }
            }
        }
        if (primo == true) {
            System.out.println("O número é primo.");
        } else {
            System.out.println("O número não é primo.");
        }
    }
}

File: ambientesJavaScript/NumPrimo.js
let num = parseInt(prompt("Digite um número para ver se é primo:"));
if (num < 2) {
    console.log("O numero não é primo.");
} else {
    let primo = true;
    for (let i = 2; i <= Math.sqrt(num); i++) {
        if (num % i === 0) {
            primo = false;
            break;
        }
    }
    if (primo) {
        console.log("O numero é primo.");
    } else {
        console.log("O número não é primo.");
    }
}